app/scripts/store/dimFarmItem.directive.js

```diff
@@ -0,0 +1,56 @@
+(function() {
+  'use strict';
+
+  angular.module('dimApp')
+        .directive('dimFarmItem', dimItem);
```

Comment:
I'd prefer if we didn't make yet another clone of `dimSimpleItem`. Can't we just set the amount and have simple item work for us?